mitmproxy/options.py

```diff
@@ -450,11 +445,7 @@ def __init__(self, **kwargs) -> None:
         self.add_option(
             "flow_detail", int, 1,
             """
-            The display detail level for flows in mitmdump: 0 (almost quiet) to 3 (very verbose).
-              0: shortened request URL, response status code, WebSocket and TCP message notifications.
-              1: full request URL with response status code
-              2: 1 + HTTP headers
-              3: 2 + full response content, content of WebSocket and TCP messages.
+            Flow display detail level in mitmdump from 0 (almost quiet) to 3 (full message contents).
```

Comment:
Didn't we just add that because nobody knew *what* is displayed at *which* level?